lib/ahoy/version.rb

```diff
@@ -1,3 +1,3 @@
 module Ahoy
-  VERSION = "1.1.1"
+  VERSION = "1.2.0.rc1"
```

Comment:
I think the maintainers should decide when to change the version number. It would make it easier for them to merge this if they disagreed with the number assigned here